src/utils/time.js

```diff
@@ -1,14 +1,18 @@
-// Build the time string manually because moment.js occasionally formats
-// the hour as 2.225071737871332e-308
-// Equivalent to `moment(date).format('h:mm A')`
-exports.formatTime = function(date) {
```

Comment:
We should keep this function and call it wherever you need to format an hour, such as `date.format('h:mm A')` or `date.format('h A')` due to the problem described by the comment. I suspect that it may be a problem with the Raspberry Pi (and related to the Skycons rendering bug), so perhaps a hardware change would eliminate the need for this hack. But unfortunately for now we'll need to continue to roll our own time formatting.

If you need to, you can change the function to accept moment dates rather than JS dates, and get individual time components using `date.get('hour')`, etc. This should hopefully be okay, as long as it's `format()` that causes the bug and not `moment(date)`.

This is what the problem looks like:

![image](https://cloud.githubusercontent.com/assets/424214/21790901/697c5576-d6ad-11e6-87aa-4eb986c33e38.png)